libs/ledger-live-common/src/exchange/providers/sell.ts

```diff
@@ -18,32 +18,23 @@ const testSellProvider: ExchangeProviderNameAndSignature = {
   version: 2,
 };
 
-let providerDataCache: Record<string, ExchangeProviderNameAndSignature> | null = null;
-
 /**
  * The result is cached after the first successful fetch to avoid redundant network calls.
  * - "fund" providers currently include Mercuryo, which is stored as a fund provider.
```

Comment:
This does not seem true anymore

Reply:
what is not true anymore? it was not set in Sara's PR anymore so better remove it

Reply:
ahhh yes removing also the comment ok got it